.github/CODEOWNERS

```diff
@@ -24,35 +24,35 @@ cirq-rigetti/**/*.* @erichulburd @kalzoo @dbanty @quantumlib/cirq-maintainers @v
 
 cirq-core/cirq/experiments/**/*.* @mrwojtek @quantumlib/cirq-maintainers @vtomole @cduck
 
-###########################################
-# docs maintainers: maintainers + @rmlarose
-###########################################
+#####################################################
+# docs maintainers: maintainers + @aasfaw + @rmlarose
+#####################################################
 
-docs/**/*.* @rmlarose @quantumlib/cirq-maintainers @vtomole @cduck
+docs/**/*.* @aasfaw @rmlarose @quantumlib/cirq-maintainers @vtomole @cduck
```

Comment:
I think you need to modify this as well:
https://github.com/quantumlib/Cirq/blob/master/dev_tools/codeowners_test.py#L22

Also, should we remove Ryan LaRose?  Is he still working on docs?

Reply:
Done. Looks like Ryan is no longer with the team, but may still be interested in contributing. Leaving for now in CODEOWNERS and will revisit at the end of this round of work.